components/Templates/class-pods_templates.php

```diff
@@ -221,6 +221,15 @@ public function add_metaboxes( $slug, $post = false ) {
 				'groups' => array(),
 			)
 		);
+		add_meta_box(
+			'pod_reference', __( 'Magic Tag Reference', 'pods' ), array(
+				$this,
+				'render_metaboxes_custom'
+			), '_pods_template', 'side', 'default', array(
+				'slug' => 'tag_reference',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 3 space(s) between "'slug'" and double arrow, but found 1.

<sub>hash: 4dcc1798e03a9e42e644225566a6026f</sub>